### PR TITLE
Rng

### PR DESCRIPTION
**Pull request by vinodkoul**
RNG tests for db820 and 410

### CODE REVIEW

File: boards/arrow,apq8096-db820c [hunk 1]
```diff
@@ -151,3 +151,7 @@ assert_device_present ufs-phy-phy1-probed ufs_qcom_phy_qmp_14nm 627000.*
 # TSENS
 assert_driver_present qcom-tsens-driver-present qcom-tsens
 assert_device_present qcom-tsens0-device-probed qcom-tsens 4a8000.*
+
+# PRNG
+assert_driver_present qcom-prng-driver-present msm-rng
+assert_device_present qcom-tsens0-device-probed msm-rng 83000.*
```
Comment (ndechesne) [inline boards/arrow,apq8096-db820c:157]: tsens0?

File: boards/qcom,apq8016-sbc [hunk 2]
```diff
@@ -107,7 +107,11 @@ assert_device_present smsm-device-probed qcom-smsm smsm
 # VFE                            0/0/0/0 (0)
 # FastRPC                        0/0/0/0 (0)
 # QCrypto                        0/0/0/0 (0)
+
 # PRNG                           0/0/0/0 (0)
+assert_driver_present qcom-prng-driver-present msm-rng
+assert_device_present qcom-tsens0-device-probed msm-rng 22000.*
```
Comment (ndechesne) [inline boards/qcom,apq8016-sbc:113]: tsens0?

File: boards/arrow,apq8096-db820c (hunk 1 above)
Comment (vinodkoul) [reply boards/arrow,apq8096-db820c:157]: oops, thanks will fix

File: boards/qcom,apq8016-sbc (hunk 2 above)
Comment (vinodkoul) [reply boards/qcom,apq8016-sbc:113]: ditto

**Comment by vinodkoul** [comment]
Verified on db820c and db410c

**Pull request closed without merging**